### PR TITLE
Install prettier

### PR DESCRIPTION
**Pull request by boyum**
This PR adds a npm run-script for checking and fixing formatting issues with Prettier. I have set a pretty standard set of rules, but these can be discussed.

The biggest issue that I see with this PR is that all its changes makes it impossible to merge the fork into Course Presentation ever again. As we have decided to go in separate directions anyways, this shouldn't be a big problem.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,4 @@
-NDLA H5P Interactive Board
-==========
+# NDLA H5P Interactive Board
```
Comment (otacke) [inline README.md:1]: Oh, does `prettier` also work on markdown? Nice. Didn't know that. (just a note :-))

**Review by otacke — CHANGES_REQUESTED** [review]
Looks all fine to me ...

... except the merge conflicts about to occur. Seems there's one commit from the master branch that your branch doesn't contain.

Just a note ...
Having single quotes in CSS is new to me, but since they can be used interchangeably and there's no H5P Group policy ... My problem :-D What's your opinion on that? If it's possible for prettier to have separate configurations for different file types (I don't know): Should we use single quotes in JS and double quotes in CSS? Or omit CSS completely? I really do not have any strong opinion here.

**Comment by boyum** [comment]
@otacke I'd prefer having double quotes everywhere. It is possible to override certain glob patterns and I've set `*.css` and `*.scss` to use double quotes now.

Also, the merge conflicts have been dealt with.

**Comment by otacke** [comment]
@boyum This is not about your taste or my taste and I will certainly not be part of any discussion about "spaces vs. tabs", "single quotes vs. double quotes", etc. I don't really care what I have to use and don't try to enforce anyone to "do it this way". Some projects prefer this, some projects prefer that. [There's a guideline that the H5P core team issued](https://h5p.org/documentation/for-developers/coding-guidelines), and NDLA wants to stick to that as closely as possible. That's all. Not everything is defined in that styleguide (e.g. the parentheses in arrow functions with one argument), so there's some ambiguity that we can solve the way we want to, but it should be coherent across all code (yes, I know, even though it's not even coherent for the code of H5P itself).

**Comment by boyum** [comment]
@otacke I agree that it's not up to you or me to decide, however I think it's important to set a project-wide setting where there aren't any set by the H5P core team. I would much rather use a code style that I didn't prefer _everywhere_ than have my preferred code style somewhere and another in the rest of the code base. I was under the impression that you wanted double quotes in CSS because those seemed most in line with the regular H5P coding style.

As prettier is a tool that can automatically fix most style problems (at least those we're discussing here), I don't think it's critical for each repository to have the same rules — prettier would fix them anyways if we were to code the wrong way. I agree that we _should_ keep the same rules everywhere, but then again, it's better to at least have a set of rules at all.

I'm really thankful for your input and I appreciate that you have gone through the official guidelines and that we can change the prettier config to fit that as closely as possible. I think we should adapt these settings for now and change them if the official guidelines were to be changed.

**Review by henriettemoe — APPROVED** [review]

**Comment by otacke** [comment]
@boyum I am not quite sure where our wires got crossed. I deliberately added "Just a note..." before my comment, said that being used to double quotes in CSS was "my problem" and asked about your opinion, but I did not request to change the rules for CSS to double quotes. The H5P core team doesn't have a rule, they themselves sometimes use single quotes, sometimes double quotes - and I really don't care. If you feel better with single quotes everywhere, let's do that. That's what my note including the question was about in the first place.

**Comment by boyum** [comment]
I'm sorry! It was not my intention to start any big discussion either. I think we're on the same side on this issue, and our team is also mostly used to double quotes in CSS, therefore I think it makes sense to keep double quotes there.

**Review by otacke — APPROVED** [review]

**Comment by otacke** [comment]
@boyum We should drink a beer together some time :-D

**Comment by boyum** [comment]
Haha yes, absolutely! I think we'd be able to solve a lot of world problems that way 💥 Have a nice weekend, Oliver!

**Pull request merged**